src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="http://127.0.0.1:5500/src/index.html" 
+    method="post"
+    class="searchbar"
+    data-qa="big"
+    >
+      <input 
+      type="text"
+      id="big" 
+      placeholder="Try &quot;Los Angeles&quot;"
+      >
```

Comment:
![image](https://user-images.githubusercontent.com/80700247/227746848-eb25921f-13a8-4d74-b265-1b89897d026c.png)

check and fix all such cases